internal/services/containers/kubernetes_cluster_data_source.go

```diff
@@ -588,6 +588,23 @@ func dataSourceKubernetesCluster() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"oidc_issuer_profile": {
+				Type:     pluginsdk.TypeList,
+				Computed: true,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"enabled": {
+							Type:     pluginsdk.TypeBool,
+							Computed: true,
+						},
+						"issuer_url": {
+							Type:     pluginsdk.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
```

Comment:
Same here, let's separate these into their own properties
```suggestion
```suggestion
			"oidc_issuer_enabled": {
				Type:     pluginsdk.TypeBool,
				Computed: true
			},
			"oidc_issuer_url": {
				Type:     pluginsdk.TypeString,
				Computed: true,
			},
```
```